src/doc/rustdoc/src/how-to-read-rustdoc.md

```diff
@@ -0,0 +1,101 @@
+# How to read rustdoc output
+
+Rustdoc's HTML output includes a friendly and useful navigation interface,
+to make it easier for users to navigate and understand your code.
+This chapter covers the major features of that interface,
+and is a great starting point for documentation authors and users alike.
+
+## Structure
+
+The `rustdoc` output is divided into three sections.
+Along the left side of each page is a quick navigation bar,
+which shows contextual information about the current entry.
+The rest of the is taken up by the search interface at the top
+and the documentation for the current item below that.
+
+## The Item Documentation
+
+The majority of the screen is taken up with the main event:
```

Comment:
"the main event"?

Reply:
English (American?) idiom meaning "the point" - think what data is to metadata. If it confused you, though, it'll confuse others, so I'll change it.

Reply:
It sounds a bit funny to me too. I'd suggest something like "The majority of the screen is taken up by the main focus:" or, even simpler, "The majority of the screen is taken up by the documentation text\[...\]".